.github/workflows/test-conda.yml

```diff
@@ -75,4 +75,4 @@ jobs:
         shell: bash
         run: |
           source activate test
-          pytest ${{ matrix.pytest_args }}
+          pytest pygeos ${{ matrix.pytest_args }}
```

Comment:
Note: you can also use the `setup.cfg` file for this: https://docs.pytest.org/en/stable/customize.html#setup-cfg